client/src/routes/Faucet.tsx

```diff
@@ -107,6 +115,20 @@ class Faucet extends PureComponent<{}, FaucetState> {
         </>
     )
 
+    private GetTokens = () => (
```

Comment:
Instead of talking about generic token, it should be more specific, like "Request OCEAN" or "Request Ocean Tokens".

Reply:
changed